lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -5,7 +5,7 @@
 <%namespace name='static' file='/static_content.html'/>
 <%inherit file="/main.html" />
 
-<%block name="bodyclass">view-teams is-in-course course js</%block>
+<%block name="bodyclass">view-teams is-view-in-course course js</%block>
```

Comment:
I was worried about lower level elements having the SMACSS version. The BEMish one is fine but since the other .view-* classes are single views I preferred the stateful class on the view. Considering that, do you have a preference?